CHANGELOG.md

```diff
@@ -12,6 +12,14 @@ As a result of this change, M3DB will allocate significantly less mmaps, but wil
 
 Operators may need to tune their kernel configuration to allow a higher number of open file descriptors. Please follow our [Kernel Configuration Guide](http://m3db.github.io/m3/operational_guide/kernel_configuration/) for more details.
 
+## New Features
+
+- **M3DB**: Add dedicated aggreagion endpoint (#1463, #1502, #1483)
+- **M3DB**: Add coordinator capability to downsample using remote m3aggregator (#1403)
```

Comment:
`M3DB` => `M3Coordinator`?

Reply:
Oops, was meant to update this to a different thing